src/awx_plugins/credentials/github_app.py

```diff
@@ -156,6 +157,29 @@ def _is_client_id(client_id_candidate: str) -> bool:
     return True
 
 
+def _is_iv1_prefix(client_id_candidate: str) -> bool:
+    """Check if it starts with Iv1 (not Iv10, Iv100, etc.)."""
+    if not client_id_candidate.startswith('Iv1'):
+        return False
+    if len(client_id_candidate) <= 3:
+        return True
+    return not client_id_candidate[3].isdigit()
+
+
+def _is_client_id(client_id_candidate: str) -> bool:
+    # Iv1: strict format with Iv1. + 16 hex chars
+    # Iv2+: just check length (19-22 chars)
+
+    if _is_iv1_prefix(client_id_candidate):
+        return _is_iv1_client_id(client_id_candidate)
```

Comment:
Why can't this just be
```suggestion
    if _is_iv1_client_id(client_id_candidate):
        return True
```
?

Reply:
Marking this thread as resolved.